WooCommerce/Classes/ViewRelated/Orders/Order Creation/OrderForm.swift

```diff
@@ -129,6 +129,26 @@ struct OrderForm: View {
     @State private var shouldShowShippingLineDetails = false
 
     var body: some View {
+        if ServiceLocator.featureFlagService.isFeatureFlagEnabled(.sideBySideViewForOrderForm) {
```

Comment:
Question: Should we expose the `EditableOrderViewModel`'s `featureFlagService` property rather than calling the `ServiceLocator` shared instance? Since this model already drives the view we can reach to it directly.

We also reach to the service locator on `OrderDetailsViewController` when checking the feature flag for handling navigation on `editOrder()`

Reply:
I don't see a particular reason to. 

The main reason to have a `featureFlagService` property, rather than directly use the singleton, is to enable unit testability. We don't unit test the view, so there's no need to do that here.

A secondary reason might be to make the line a bit shorter/easier to read, but if we reach in to the view model, it won't be much shorter.